examples/standalone_mode_demo.ipynb

```diff
@@ -0,0 +1,609 @@
+     ]
+    }
+   ],
+   "source": [
+    "# we can restore the model from the checkpoint:\n",
+    "model.restore_model(best_model_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:Start to load partition from /mnt/efs/gsf-data/movielen_100k_train_val_1p_4t/part0/graph.dgl which is 5839353 bytes. It may take non-trivial time for large partition.\n",
+      "INFO:root:Finished loading partition.\n",
+      "INFO:root:Finished loading node data.\n",
+      "INFO:root:Finished loading edge data.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a dataset for inference, we use the same MovieLens graph\n",
+    "infer_data = GSgnnNodeInferData(\n",
+    "    config.graph_name, \n",
+    "    config.part_config,\n",
+    "    eval_ntypes=config.target_ntype,\n",
+    "    node_feat_field=config.node_feat_name,\n",
+    "    label_field=config.label_field)\n",
+    "\n",
+    "# Set up dataloader for the inference dataset\n",
+    "infer_dataloader = GSgnnNodeDataLoader(\n",
+    "    infer_data, infer_data.test_idxs, fanout=config.fanout, \n",
+    "    batch_size=100, device=device,\n",
+    "    train_task=False)\n",
+    "\n",
+    "\n",
+    "# Create an Inferrer object\n",
+    "infer = GSgnnNodePredictionInferrer(model)\n",
+    "infer.setup_device(device=device)\n",
+    "infer.setup_evaluator(evaluator)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:root:Writing GNN embeddings to nc_model/epoch-2/infer/embeddings in pytorch format.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run inference on the inference dataset\n",
+    "infer.infer(infer_dataloader, \n",
+    "            save_embed_path=os.path.join(best_model_path, 'infer/embeddings'),\n",
+    "            save_prediction_path=os.path.join(best_model_path, 'infer/predictions'),\n",
+    "            use_mini_batch_infer=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 848K\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu   40 Oct  5 20:47 emb_info.json\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu 842K Oct  5 20:47 movie_emb.part00000.bin\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The GNN embeddings on the inference graph are saved to:\n",
+    "!ls -lh {best_model_path}/infer/embeddings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 16K\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu 14K Oct  5 20:47 predict-00000.pt\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
We are changing the output of prediction. Please talk to Xiang for the new output format.

Reply:
The new prediction output will be updated.